MdeModulePkg/Core/Dxe/Misc/MemoryProtection.c

```diff
@@ -1300,8 +1319,10 @@ ApplyMemoryProtectionPolicy (
   UINT64  OldAttributes;
   UINT64  NewAttributes;
 
+  // DEBUG ((DEBUG_INFO, "ApplyMemoryProtectionPolicy: 0x%016lx - 0x%016lx\n", Memory, Length));

```

Comment:
Is this commented out now because the call occurs so much? Can we add a MU_CHANGE here?

Reply:
I must have added this print when testing and forgotten about it -- removed